packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/media-state-controls.js

```diff
@@ -14,7 +14,7 @@ import {skipInFirefox} from '@ciscospark/test-helper-mocha';
 
 import {boolToStatus} from '../..';
 
-function assertLocusMediaState(call, {
+export function assertLocusMediaState(call, {
```

Comment:
If we're going to use this in multiple files, let's move it up to the `lib` directory.